src/RandomStream.java

```diff
@@ -0,0 +1,40 @@
+import java.util.function.IntSupplier;
+import java.util.stream.IntStream;
+
+public class RandomStream  {
+
+	public static IntStream pseudoRandomStream(int seed) {
```

Comment:
what about this method?

Reply:
He is so named on stepik.

Reply:
Also corrected all according to the comments.

Reply:
try to simplify this method
we can create implementation in one line

Reply:
fixed